exist-core/src/main/java/org/exist/xquery/DynamicCardinalityCheck.java

```diff
@@ -72,16 +72,16 @@ public Sequence eval(Sequence contextSequence, Item contextItem) throws XPathExc
                 "CONTEXT ITEM", contextItem.toSequence());}
         }
         final Sequence seq = expression.eval(contextSequence, contextItem);
-        int actualCardinality;
+        Cardinality actualCardinality;
         if (seq.isEmpty())
-            {actualCardinality = Cardinality.EMPTY;}
+            {actualCardinality = Cardinality.EMPTY_SEQUENCE;}
```

Comment:
formatting :-(

Reply:
Sure, but sorry, I am not going to fix all existing formatting!